server/src/projects/projects.service.ts

```diff
@@ -39,9 +39,12 @@ export class ProjectsService {
       `);
       return this.overwriteCodesWithLabels(records);
     } catch(e) {
-      const errorMessage = `Error finding projects by contact ID. ${e.message}`;
+      const errorMessage = `Error finding projects by contact ID. CRM: ${e.message}`;
```

Comment:
The promise above could fail for other reasons — it's not necessarily an error that originates from the CRM system itself.

Reply:
dah, true... hmm.